contracts/cw721-base/src/state.rs

```diff
@@ -44,3 +44,10 @@ pub fn increment_tokens<S: Storage>(storage: &mut S) -> StdResult<u64> {
     TOKEN_COUNT.save(storage, &val)?;
     Ok(val)
 }
+
+// indexed map needs function, not const (for now at least)
```

Comment:
I guess this is because of the lambda, right?

Reply:
Yes, you cannot build vec structs with const. It must all be in the stack and computer at compile time.

In the index refactor (using structs instead of this), I could theoretically create the structure as a const (and I manually created an IndexMap struct as const in a branch), but the compiler did not let me use `MultiIndex::new` as it took a function pointer as an argument - even if I declare that function pointer as a `const` (which let's me build a struct), there is no way to declare that in `MultiIndex::new` signature. I spent like 2 hours in the const rabbit hole...

Reply:
:-) I can imagine.